Project.toml

```diff
@@ -2,15 +2,15 @@ name = "PowerSystemsUnits"
 uuid = "c279aad7-7dc9-58d3-995b-109d3f5deb94"
 license = "MIT"
 author = "Invenia Technical Computing"
-version = "0.2.0"
+version = "0.2.1"
 
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
-Unitful = "0.15, 1.6"
+Unitful = "0.15 - 1.6"
```

Comment:
Do we ever use this syntax? I want to keep the compat to 1.6 max but also support past versions of it.

Another option is to do "0.15, ~1.6" but this is more restrictive

Reply:
Oh it doesn't work for Julia 1.0